newsrevenuehub/rev-engine
# [NRH-174] Pages list

#### What's this PR do?
Here we create a styled list of pages, organized by revenue program

#### How should this be manually tested?
- As an OrgAdmin, visit the dashboard and navigate to "Pages"
- Observe that there is a list of pages, organized by RevProgram (the RevProgram accordions are collapsible)
- Each Page card should contain a thumbnail of the the page as it was last saved, or filled text if not available
- The pages should be sorted by published_date, with most recent publishes first.
- Live pages should have a small green icon indicating they are live.
- Clicking on a page will take you to the detail view (edit interface)

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
No
